src/main/kotlin/model/ManageRecalls.kt

```diff
@@ -0,0 +1,103 @@
+package model
+
+import com.structurizr.model.Container
+import com.structurizr.model.Model
+import com.structurizr.model.Person
+import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.AutomaticLayout
+import com.structurizr.view.ViewSet
+import uk.gov.justice.hmpps.architecture.CloudPlatform
+import uk.gov.justice.hmpps.architecture.HMPPSAuth
+import uk.gov.justice.hmpps.architecture.HMPPSSoftwareSystem
+import uk.gov.justice.hmpps.architecture.NOMIS
+import uk.gov.justice.hmpps.architecture.annotations.Tags
+
+class ManageRecalls private constructor() {
+  companion object : HMPPSSoftwareSystem {
+    lateinit var system: SoftwareSystem
+    lateinit var ppcsCaseWorker: Person
+    lateinit var manageRecallsUiContainer: Container
+    lateinit var manageRecallsApiContainer: Container
+    lateinit var dbContainer: Container
+    lateinit var prisonsRegisterContainer: Container
+
+    override fun defineModelEntities(model: Model) {
+
+      system = model.addSoftwareSystem(
+        "Manage Recalls",
+        "The case management system for recalls"
+      )
+
+      dbContainer = system.addContainer(
+        "Manage Recalls API Database",
+        "Database to store recalls caseworking info",
+        "RDS Postgres DB"
+      ).apply {
+        Tags.DATABASE.addTo(this)
+        CloudPlatform.rds.add(this)
+      }
+
+      prisonsRegisterContainer = system.addContainer(
+        "Prisons Register",
```

Comment:
🙋‍♂️ Probably worth adding as a separate thing, as we have [CourtRegister](https://github.com/ministryofjustice/hmpps-architecture-as-code/blob/17570ed350ea7bbdedb665d98325c95c6b68611c/src/main/kotlin/model/CourtRegister.kt) too

Reply:
gotcha - thanks for the nudge!